File: core/src/org/rexcellentgames/burningknight/entity/item/accessory/equippable/Zoom.java
package org.rexcellentgames.burningknight.entity.item.accessory.equippable;

import org.rexcellentgames.burningknight.entity.creature.player.Player;

public class Zoom extends Equippable {
	@Override
	public void onEquip(boolean load) {
		super.onEquip(load);
		Player.seeMore = true;
	}

	@Override
	public void onUnequip(boolean load) {
		super.onUnequip(load);
		Player.seeMore = false;
	}
}

File: core/src/org/rexcellentgames/burningknight/entity/item/accessory/equippable/Zoom.kt
package org.rexcellentgames.burningknight.entity.item.accessory.equippable

import org.rexcellentgames.burningknight.entity.creature.player.Player

class Zoom : Equippable() {
	override fun onEquip(load: Boolean) {
		super.onEquip(load)
		Player.seeMore = true
	}

	override fun onUnequip(load: Boolean) {
		super.onUnequip(load)
		Player.seeMore = false
	}

	override fun canBeUpgraded(): Boolean {
		return false
	}

	override fun canBeDegraded(): Boolean {
		return false
	}
}